.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: dart
 
 dart:
-  - 2.2.0
+  - 2.1.0-dev
```

Comment:
you need to put in an EXPLICIT dart version that we've shipped!

Reply:
sorry was trying to track it down. Pushed the commit prematurely. Using 2.1.2-dev.0.0.